receiver/dockerstatsreceiver/metadata.yaml

```diff
@@ -628,3 +628,27 @@ metrics:
       aggregation: cumulative
     attributes:
       - interface
+
+  # Inspect
+  container.cpu.limit:
+    enabled: false
+    description: "CPU limit set for the container."
+    unit: "1"
+    gauge:
+      value_type: double
+  container.cpu.shares:
+    enabled: false
+    description: "CPU shares set for the container."
+    unit: "1"
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+  container.restart.count:
```

Comment:
See https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/21803#issuecomment-1554304700

Reply:
Hi @astencel-sumo 

Thanks for spotting this.
This is not a namespace so it makes all sense to remove the .count. I have submitted the fix (https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/21803/commits/3d242fbe92ea61d37cb41090ea81831241a8c240)